var/spack/repos/builtin/packages/vtkm/package.py

```diff
@@ -66,6 +68,12 @@ def cmake_args(self):
             else:
                 options.append("-DVTKm_ENABLE_CUDA:BOOL=OFF")
 
+            # double precision
+            if "+doubleprecision" in spec:
```

Comment:
@HaochengLIU the names of the variants appear to be swapped with the CMake options they enable, is that intended?

Also, originally these were both disabled automatically. Is there a reason to make a separate Spack package variant for each CMake option? It's not required but if you think a user would always want these enabled or disabled together, you could use a single package variant for that).

Reply:
@scheibelp Thanks for pointing the swapped name out. It's a typo. Fixed.

Originally by default double precision is enabled while 64 bit ids is disabled. This commit repects the original value. These two options should be seperated since they refer to different settings in VTK-m.